Source/Core/Core/Boot/Boot.cpp

```diff
@@ -151,30 +152,53 @@ bool CBoot::LoadMapFromFilename()
 // It does not initialize the hardware or anything else like BS1 does.
 bool CBoot::Load_BS2(const std::string& _rBootROMFilename)
 {
-	const u32 USA = 0x1FCE3FD6;
+	/* Adler32 Hashes
+	const u32 USA_v1_0 = 0x1FCE3FD6;
 	const u32 USA_v1_1 = 0x4D5935D1;
-	const u32 JAP = 0x87424396;
-	const u32 PAL = 0xA0EA7341;
+	const u32 USA_v1_2 = 0xDF6CF65B;
+	const u32 JAP_v1_0 = 0xB2FCDB25;
+	const u32 JAP_v1_1 = 0x46E899DB;
+	const u32 JAP_unknown = 0x87424396;
+	const u32 PAL_v1_0 = 0xAF912C43;
+	const u32 PAL_unknown = 0xA0EA7341;
```

Comment:
Should I also remove the Panasonic ones? They were already commented out before I changed anything.